docker/registry/protoc-base/gradle.properties

```diff
@@ -1,5 +1,3 @@
 io.deephaven.project.ProjectType=DOCKER_REGISTRY
 deephaven.registry.imageName=ghcr.io/deephaven/protoc-base:latest
-deephaven.registry.imageId=ghcr.io/deephaven/protoc-base@sha256:dcbde68539a04be744963d1f3c90c30be1c7f75d291635d2f1efc3ee0c0167ed
-# TODO(deephaven-base-images#55): arm64 native image for protoc-base
-deephaven.registry.platform=linux/amd64
```

Comment:
Sorry, this hasn't been incorporated into the update script. Please make sure this doesn't get removed.